wgsl/index.bs

```diff
@@ -442,6 +442,18 @@ literal_or_ident
     and [=fragment=] inputs.
     See [[#interpolation]].
 
+  <tr><td><dfn noexport dfn-for="attribute">`invariant`
+    <td>*None*
+    <td>Must only be applied to an entry point return type, or member of a
+    [=structure=] type.
+    Must not be applied to built-in variables.
```

Comment:
This is the opposite of what we agreed so far.
In Metal, invariant can *only* be applied to the output position from VS, nothing else.

Reply:
Wow, I really misread that issue. I'll update this to just position.